docs/STAR_2.7.11b_DIPLOID.md

```diff
@@ -0,0 +1,119 @@
+# Kids First Kids First STAR Diploid Beta
+
+This is an alternative alignment and quantification method currently in **_beta phase_**.
+It uses a patient's DNA variant calls to create a "personal genome" (PG) for improved alignment.
+It is purported to have fewer multi-mapping/better unique mapping for potentially improved gene and isoform level quantification.
+It cannot be used in fusion calling
+The STAR Diploid mode has a known bug with an unknown cause manifests as a seg fault for up to 20% of normal sample inputs and 100% of tumor inputs. 
+
+![data service logo](https://github.com/d3b-center/d3b-research-workflows/raw/master/doc/kfdrc-logo-sm.png)
+
+## Introduction
+This pipeline runs the following steps:
+1. STAR Genome Generate per individual, or will skip if one has been generated already.
+    -  Strip existing annotations as defined by the user. Since STAR requires an uncompressed vcf, this helps make the input file size smaller as only variant calls and `FORMAT` info are needed
```

Comment:
```suggestion
    -  Strip existing annotations as defined by the user. Since STAR requires an uncompressed VCF, this helps make the input file size smaller as only variant calls and `FORMAT` info are needed
```